src/frontend/pages/Checkout/Checkout.jsx

```diff
@@ -0,0 +1,150 @@
+import React from "react";
+import { useCart, useProduct, useWishlist } from "../../contexts/hooks-export";
+import { Link } from "react-router-dom";
+import "./checkout.css";
+import { ToastContainer } from "react-toastify";
+
+export const Checkout = () => {
+  const { filteredData } = useProduct();
+  const { wishlistState, addToWishlist } = useWishlist();
+  const { wishlistData } = wishlistState;
+  const { cartState, removeFromCart, handleCartQuantity } = useCart();
+  const { cartData } = cartState;
+
+  const totalDiscount = cartData.reduce(
+    (totalDiscount, cartItem) =>
+      totalDiscount +
+      Math.round((cartItem.price * cartItem.qty * cartItem.discount) / 100),
+    0
+  );
+  const totalPrice = cartData.reduce(
+    (totalPrice, cartItem) => totalPrice + cartItem.price * cartItem.qty,
+    0
+  );
+  const deliveryCharge = 100;
+
+  return (
+    <div>
+      <h1 class="text-center">Checkout</h1>
+      <div class="underline"></div>
+      <div class="flex cartitems-container mx-auto">
+        <div class="card p-1 align-self-start">
+          <h3 class="pb-1">Price Details</h3>
+          <hr />
+          <div class="flex space-between pt-1">
+            <p>Price ({cartData.length} items)</p>
+            <p>&#8377; {totalPrice}</p>
+          </div>
+          <div class="flex space-between pt-1">
+            <p>Discount</p>
+            <p>- &#8377;{totalDiscount}</p>
+          </div>
+          <div class="flex space-between pb-1 pt-1">
```

Comment:
You can change class to className here.

Reply:
yeah will change it thanks for pointing it out